cloudops.go

```diff
@@ -48,6 +48,9 @@ type Compute interface {
 	// InspectInstanceGroupForInstance inspects the instance group to which the
 	// cloud instance with given ID belongs
 	InspectInstanceGroupForInstance(instanceID string) (*InstanceGroupInfo, error)
```

Comment:
Can u also add support in the InspectInstanceGroupForInstance API to return current size. Just add the GKE implementation for now. We will add KOPS when we start on the KOPS effort.